.travis.yml

```diff
@@ -39,6 +40,7 @@ addons:
     - ffmpeg
 install:
   - pip install --upgrade pip
+  - pip install git+git://github.com/okor/remotecv.git@439324abb8014969467ad18c65188b8e3db48b78
```

Comment:
This is a temporary hack until a new version of `remotecv` with Pillow version 6 is released. After the new version is released we can remove this instruction.